repo/github_release.go

```diff
@@ -25,14 +23,7 @@ const (
 	ISO8601 = "20060102T150405Z0700"
 )
 
-var httpClient = &http.Client{
-	Timeout: 30 * time.Second,
-}
-
-var (
-	_ registory.Registory = (*GithubRelease)(nil)
-	_ storage.Fetcher     = (*GithubRelease)(nil)
-)
+var _ registory.Registory = (*GithubRelease)(nil)
```

Comment:
What do you mean this line for syntax?

Reply:
To ensure at compile time that GithubRelease satisfies the registry.Registry.

On another note, I'm typoing...

Reply:
I see, it's a type assertion to an interface.